modelkit/utils/cache.py

```diff
@@ -16,7 +19,30 @@ class CacheItem(NamedTuple, Generic[ItemType]):
     missing: bool = True
 
 
-class RedisCache:
+class Cache(abc.ABC):
+    @staticmethod
+    def from_settings(settings):
+        if settings.cache_provider == "redis":
+            return RedisCache(settings.settings.host, settings.settings.port)
+        if settings.cache_provider == "native":
+            return RedisCache(
```

Comment:
NativeCache?

Reply:
Certainly! I wonder what I've been testing this whole time 😬